app/models/migration/induction_record.rb

```diff
@@ -6,5 +6,11 @@ class InductionRecord < Migration::Base
     belongs_to :mentor_profile, class_name: "Migration::ParticipantProfile"
     belongs_to :schedule
     belongs_to :preferred_identity, class_name: "Migration::ParticipantIdentity"
+
+    has_one :school_cohort, through: :induction_programme
+    has_one :school, through: :school_cohort
+    has_one :partnership, through: :induction_programme
+    has_one :delivery_partner, through: :partnership
+    has_one :lead_provider, through: :partnership
```

Comment:
Can we add specs for these?

Reply:
The test strategy for migration models is a bit unclear, and coverage is pretty minimal. I think that's ok for now since these models mainly just fetch ECF records and a few associations - there's not much business logic involved, so the risk is likely low.

Nonetheless, I have added some tests to ensure coverage for this model, and we can follow up if we feel we need better overall coverage.